src/Symfony/Component/HttpFoundation/Session/Storage/NativeSessionStorage.php

```diff
@@ -219,6 +219,12 @@ public function regenerate($destroy = false, $lifetime = null)
      */
     public function save()
     {
+        // In PHP <7.2 session_write_close() does not error if the session is
```

Comment:
maybe this should be written as `In PHP >=7.2 session_write_close() errors if the session is not started?`

Reply:
I was trying to say that this is following the logic of session_write_close in PHP <7.2 - in PHP >=7.2 it returns FALSE.

However I've just read the docs on the interface which say
```
    /**
     * Force the session to be saved and closed.
     *
     * This method must invoke session_write_close() unless this interface is
     * used for a storage object design for unit or functional testing where
     * a real PHP session would interfere with testing, in which case
     * it should actually persist the session data if required.
     *
     * @throws \RuntimeException if the session is saved without being started, or if the session
     *                           is already closed
     */
    public function save();
```

So now I'm not sure what's the right thing to do. What's clear is that is not what \Symfony\Component\HttpFoundation\Session\Storage\NativeSessionStorage::save() does in 3.4 HEAD.